### ISSUE TITLE
ToolTip.StaysOpen does not work?

### ISSUE THREAD

**Issue by akimusume** [dotnet/wpf#4624]
* .NET Core Version: Net 5
* Windows version: 19043
* Does the bug reproduce also in WPF for .NET Framework 4.8?:  Yes in 4.5

 
 
 **Problem description:**
The ToolTip.StaysOpen property does not work like Popup.StaysOpen. How can I keep the ToolTip open until I click on other control?

 **Actual behavior:** 
The ToolTip disappeared after I move the mouse over other control without click.

 **Expected behavior:**
 When ToolTip.StaysOpen is true, it should keep open until I click on other control.

**Comment by SamBent** [comment]
>How can I keep the ToolTip open until I click on other control?

You can't.  The rules for when to dismiss a ToolTip are different from the rules for a Popup, and they don't involve StaysOpen.  This is by design, motivated by accessibility standards and by a general principle that tooltips should not affect the application's behavior in any way other than simply displaying their content.

However, the rules have changed in .NET 6.0, in response to new accessibility standards.  Details [here](https://github.com/dotnet/wpf/blob/main/src/Microsoft.DotNet.Wpf/specs/tooltip.md).  The new rules keep ToolTips open longer than before, so perhaps they meet your needs.   But the accessibility standards still require that moving the mouse sufficiently far away closes the tooltip (without a click);  doing literally what you asked for would violate the standards.
